Makefile

```diff
@@ -32,70 +32,46 @@ bin/linuxkit: $(LINUXKIT_DEPS) | bin
 	rm tmp_linuxkit_bin.tar
 	touch $@
 
-test-initrd.img: $(MOBY) test/test.yml
-	$(MOBY) build test/test.yml
-
-test-bzImage: test-initrd.img
-
-.PHONY: test-qemu-efi
-test-qemu-efi: $(LINUXKIT) test-efi.iso
-	$(LINUXKIT) run $^ | tee test-efi.log
-	$(call check_test_log, test-efi.log)
-
 bin:
 	mkdir -p $@
 
 install:
 	cp -R ./bin/* $(PREFIX)/bin
 
-define check_test_log
-	@cat $1 |grep -q 'test suite PASSED'
-endef
-
-.PHONY: test-hyperkit
-test-hyperkit: $(LINUXKIT) test-initrd.img test-bzImage test-cmdline
-	rm -f disk.img
-	$(LINUXKIT) run test | tee test.log
-	$(call check_test_log, test.log)
-
-.PHONY: test-gcp
-test-gcp: $(LINUXKIT) test.img.tar.gz
-	$(LINUXKIT) push gcp test.img.tar.gz
-	$(LINUXKIT) run gcp test | tee test-gcp.log
-	$(call check_test_log, test-gcp.log)
-
 .PHONY: test
-test: $(LINUXKIT) test-initrd.img test-bzImage test-cmdline
-	$(LINUXKIT) run test | tee test.log
-	$(call check_test_log, test.log)
+test: $(LINUXKIT) $(MOBY)
+	@MOBY=$$(if [ "$(MOBY)" = "bin/moby" ]; then echo $(PWD)/$(MOBY); else echo $(MOBY); fi) \
```

Comment:
Does that mean that we can assume `moby` or `linuxkit` are in the `$PATH`?
That would simplify things a lot!